Exercises/Classwork/classwork.js

```diff
@@ -0,0 +1,18 @@
+// https://gist.github.com/Mike-Savin/27922f83c368fd1cf636b65abb760874
+// task4.js
+
+function generateUpToMatch(expectedArray, generatedArrayLength) {
```

Comment:
The point of this task is to make you to compare two arrays element by element.
You need to generate whole result array randomly, not a part of it. And you need to regenerate it randomly until it last `n` elements will equal to expected array.

Now you generate only part of result randomly, and then just add expected array element to the end, which is not correct